src/main/java/com/digital/home/controller/SalaryController.java

```diff
@@ -0,0 +1,20 @@
+package com.digital.home.controller;
+
+import com.digital.home.model.DailyTransaction;
+import com.digital.home.model.SalaryDetails;
+import com.digital.home.service.SalaryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class SalaryController {
+
+    @Autowired
+    private SalaryService salaryService;
+
+    @GetMapping("/salary/{projectId}")
```

Comment:
Add 2 more arguments fromDate and toDate and use it in the query. Sometimes the salary may be daily or weekly

Reply:
Added "fromDate" and "toDate"